src/screens/PluginStore/components/tests/PluginList.test.tsx

```
@@ -0,0 +1,245 @@
+// src/screens/PluginStore/components/tests/PluginList.test.tsx
+
+import React from 'react';
+import { render, screen, fireEvent } from '@testing-library/react';
+import { vi, describe, it, expect } from 'vitest';
+import PluginList from '../PluginList';
+import type { IPluginMeta } from 'plugin';
+import { plugins } from 'chart.js';
```

ESLint

[error] 'plugins' is defined but never used.

(@typescript-eslint/no-unused-vars)